geotrellis/geotrellis-server
# Add authorization to services

Overview
-----

This PR adds simple authorization with JWT or a few other strategies to services. It includes debug
output to verify that authorization works. In practice, we'd expand on configurability in the auth keys
to let users specify their own flows for obtaining an authorization workflow.

Testing
-----

- create a random signing key
- `export SIGNING_KEY=thatkey`
- run the server
- copy the jwt it offers you
- make an authed request to `/ping`
- it should succeed
- make an unauthed request to `/ping`
- it should fail

Closes #20 

Closes #16
